debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ObjectStructureFormatter.java

```diff
@@ -238,6 +228,33 @@ private void formatArrayObject(PrintStream out, J9ClassPointer localClazz, U8Poi
 		}
 	}
 
+	private void formatReferenceObject(PrintStream out, int tabLevel, J9ClassPointer localClazz, int begin, int finish, J9IndexableObjectPointer array) throws CorruptDataException
```

Comment:
might be better to make methods, formatReferenceObjectArray and formatFlattenedObjectArray